content/docs/reactivesearch/v3/advanced/stateprovider.md

```diff
@@ -113,6 +113,23 @@ Here is a list of all the valid keys:
 -   **error** returns the error
 -   **hits** An array of hits obtained from the applied query.
 -   **aggregations** An array of aggregations obtained from the applied query.
+-   **promotedData** An array of promoted results obtained from the applied query. [Read More](/docs/search/Rules#part-1-introduction).
+-   **resultStats**: `object`
+    An object with the following properties which can be helpful to render custom stats:
+    -   **`numberOfResults`**: `number`
+        Total number of results found
+    -   **`numberOfPages`**: `number`
+        Total number of pages found based on current page size
+    -   **`currentPage`**: `number`
```

Comment:
@ShahAnuj2610 Only add properties which we really provide, take care before copying things.
We don't provide `currentPage` & `numberOfPages`, update it everywhere.

Reply:
@bietkul updated